src/main/kotlin/no/nav/familie/ef/søknad/api/dto/søknadsdialog/Medlemskap.kt

```diff
@@ -1,4 +1,22 @@
 package no.nav.familie.ef.søknad.api.dto.søknadsdialog
 
-data class Medlemskap(val søkerBosattINorgeSisteTreÅr: BooleanFelt? = null,
-                      val søkerOppholderSegINorge: BooleanFelt? = null)
+data class Medlemskap(
+        val perioderBoddIUtlandet: List<PerioderBoddIUtlandet>,
+        val søkerBosattINorgeSisteTreÅr: BooleanFelt,
```

Comment:
Dette feltet heter bosattNorgeSisteÅrene, for å slippe å måtte endre variabelnavn hvis regelen endres til to eller fem år. Burde vært vurdert her også.

Reply:
Kan godt fikse det også.

Reply:
...men det blir en refaktorering også i frontend, så jeg venter venter med dette til frontend har kapasitet.